scripts/shelf-pack/2.0.0/script.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+MASON_NAME=shelf-pack
+MASON_VERSION=2.0.0
+MASON_HEADER_ONLY=true
+
+. ${MASON_DIR}/mason.sh
+
+function mason_load_source {
+    mason_download \
+    https://github.com/mapbox/shelf-pack-cpp/archive/v${MASON_VERSION}.tar.gz \
+    443ba2e07a157393d50f18cd19ca9c428000265e
+    mason_extract_tar_gz
+
+    export MASON_BUILD_PATH=${MASON_ROOT}/.build/shelf-pack-cpp-${MASON_VERSION}
+}
+
+function mason_compile {
+    mkdir -p ${MASON_PREFIX}/include/mapbox
+    cp -v include/*.hpp ${MASON_PREFIX}/include/mapbox
+    cp -v README.md LICENSE.md ${MASON_PREFIX}
+}
```

Comment:
A simpler approach to copying headers is https://github.com/mapbox/mason/blob/master/scripts/geometry/0.9.1/script.sh#L20-L21